src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -103,7 +107,15 @@ public Command parseCommand(String userInput) throws ParseException {
             }
 
         case ListCommand.COMMAND_WORD:
```

Comment:
The existing structure looks good to me. What do you think about the possibility of having a ListCommandParser to handle which specific type of list command to create instead of doing it in the AddressBookParser?